.gitignore

```diff
@@ -18,3 +18,5 @@
 /public/government-frontend
 /test/wraith/wip*
 shots/*
+
+.vscode
```

Comment:
You could add this to your global git config asa global ignore so you don't have to add this to each repo you work on. We try to avoid editor specific files in the .gitignore file.